turbo.json

```diff
@@ -2,33 +2,62 @@
   "$schema": "https://turbo.build/schema.json",
   "globalDependencies": ["**/.env.*local"],
   "globalEnv": ["DEV"],
-  "pipeline": {
-    "//#clean:vitest-mjs": { "cache": false },
-    "//#format:prettier": { "cache": false },
-    "//#format:syncpack": { "cache": false },
-    "//#lint:prettier": { "cache": false },
-    "//#lint:syncpack": { "cache": false },
+  "tasks": {
+    "//#clean:vitest-mjs": {
+      "cache": false
+    },
+    "//#format:prettier": {
+      "cache": false
+    },
+    "//#format:syncpack": {
+      "cache": false
+    },
+    "//#lint:prettier": {
+      "cache": false
+    },
+    "//#lint:syncpack": {
+      "cache": false
+    },
     "build": {
       "dependsOn": ["compile", "proto", "^build"],
-      "dotEnv": [".env"],
```

Comment:
New lint caught an outdated variable here. All other changes in this file were due to auto-formatting.